greentest/2.7/test_ssl.py

```diff
@@ -54,6 +61,17 @@ def test_sslwrap_simple(self):
 # Issue #9415: Ubuntu hijacks their OpenSSL and forcefully disables SSLv2
 def skip_if_broken_ubuntu_ssl(func):
     if hasattr(ssl, 'PROTOCOL_SSLv2'):
+        if hasattr(ssl, 'SSLContext'): # >= 2.7.9
+            def f(*args, **kwargs):
+                try:
+                    ssl.SSLContext(ssl.PROTOCOL_SSLv2)
+                except ssl.SSLError:
+                    if (ssl.OPENSSL_VERSION_INFO == (0, 9, 8, 15, 15) and
```

Comment:
Perhaps the thinking is that there's some benefit in keeping the changes between gevent's forked tests and the original tests as minimal as possible (after all, the 'skip' function already existed, it just got updated). That way, the thinking probably goes, merges are easier going forward. (True, merges got more difficult since the forked test cases now run in either <=2.7.8 more or >=2.7.9 mode, but eventually once support for 2.7.9 is required, it can be simplified again.) Plus, removing the decorator would have a follow-on effect that would introduce *more* unrelated changes to the test file.